modules/memory/README.md

```diff
@@ -0,0 +1,80 @@
+# terraform-datadog-monitor
```

Comment:
Seems like we should change this to reflect the specific module, not the repo. Noticed this in the other modules as well.